Comanche055/FRESH_START_AND_RESTART.agc

```diff
@@ -1281,29 +1288,29 @@ FCADRMM1	EQUALS
 		FCADR	P06
 		FCADR	GTSCPSS1			# GYROCOMPASS STANDARD LEAD IN.
 
-# THE PREMM TABLE CONTAINS THE E-BANK, MAJOR MODE, AND PRIORITY
+# THE PREMM TABLE CONTAINS THE E-BANK,MAJOR MODE, AND PRIORITY
 # INFORMATION, IT IS IN THE FOLLOWING FORM,
-#
+
 #	PPP PPE EEM MMM MMM
 #
 # Page 207
-#	WHERE THE	7 M BITS CONTAIN THE MAJOR MODE NUMBER
-#			3 E BITS CONTAIN THE E-BANK NUMBER
-#			5 P BITS CONTAIN THE PRIORITY AT WHICH THE JOB IS
-#			    TO BE STARTED
-#
+#	WHERE THE 7 M BITS CONTAIN THE MAJOR MODE NUMBER
+#		   3 E BITS CONTAIN THE E-BANK NUMBER
+#		   5 P BITS CONTAIN THE PRIORITY AT WHICH THE JOB IS
+#			TO BE STARTED
+
 #	FOR EXAMPLE,
-#
+
 #		PREMM1		OCT	67213		# PRIORITY	33
-#							# E-BANK	5
+#							# E-BANK	 5
 #							# MAJOR MODE	11
-#				OCT	25437		# PRIORITY	12
-#							# E-BANK	6
+#				OCT	25437		# PRIORITY	 12
+#							# E-BANK	 6
 #							# MAJOR MODE	31
 
 PREMM1		EQUALS
 		OCT	27117		# MM 79		EBANK 4		PRIO 13
-		OCT	27116		# MM 78		EBANK 4		PRIO 13 (original says PR23 13)
```

Comment:
Again, here, I gotta change the code.

> (original says PR23 13)

cc: @oldmud0

Reply:
Not sure what that means.